client/app/pages/queries/QueryView.jsx

```diff
@@ -35,18 +37,10 @@ import useDeleteVisualization from "./hooks/useDeleteVisualization";
 
 import "./QueryView.less";
 
-// ANGULAR_REMOVE_ME: Update with new Router
-function updateUrlSearch(...params) {}
-
 function useFullscreenHandler(available) {
-  // TODO: bring back URL sync
   const [fullscreen, setFullscreen] = useState(false);
   const toggleFullscreen = useCallback(() => setFullscreen(!fullscreen), [fullscreen]);
 
-  useEffect(() => {
-    updateUrlSearch("fullscreen", fullscreen ? true : null);
-  }, [fullscreen]);
```

Comment:
I removed the sync w/ URL on purpose. I'm not sure we need it here.

Reply:
Yeah, I actually kept it thinking about sharing the URL and to have the same behavior the Dashboard page have. But both cases are not that important anyway.